go.mod

```diff
@@ -26,9 +25,12 @@ require (
 	github.com/tektoncd/pipeline v0.10.1
 	github.com/tektoncd/plumbing v0.0.0-20191218171343-56a836c50336
 	github.com/tektoncd/triggers v0.2.1-preview.0.20200130181252-68f571334722
+	go.opencensus.io v0.22.1
 	go.uber.org/multierr v1.1.0
 	golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
+	golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898
```

Comment:
https://godoc.org/golang.org/x/xerrors

> These functions were incorporated into the standard library's errors package in Go 1.13

Reply:
@ppitonak indeed (with a slight signature change). Anyway, we can get rid of those in a follow-up (also, depending on our dependency it might become a transitive dep anway)